app/assets/stylesheets/pregnancies.scss

```diff
@@ -10,6 +10,12 @@ $font-size: 17px;
   padding: 15px 0;
 }
 
+.lmp-2{
```

Comment:
Should we adjust this class name to reflect what it's actually doing? Maybe `hide-label`?